Doc/library/locale.rst

```diff
@@ -423,10 +423,13 @@ The :mod:`locale` module defines the following exception and functions:
     .. versionadded:: 3.5
 
 
-.. function:: atof(string)
+.. function:: atof(string, func=float)
 
-   Converts a string to a floating point number, following the :const:`LC_NUMERIC`
-   settings.
+   Converts a string to a number, following the :const:`LC_NUMERIC` settings,
+   by calling *func* on the result of calling :func:`delocalize` on *string*.
+
+   .. versionchanged:: 3.5
```

Comment:
Looking back through the `git blame`, that parameter was added over 20 years ago.  It looks like the function was moved in the commit that you referenced.

Reply:
Good catch!  Apparently I wasn't looking closely enough.  Apparently `atof(str, func)` was present when the `locale` module was introduced in eef1d4e8b16.  I'll remove `.. versionchanged`.